aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -431,11 +435,8 @@ impl BlockAptosVM {
 
                 Ok(BlockOutput::new(output_vec))
             },
-            Err(BlockExecutionError::FallbackToSequential(e)) => {
-                unreachable!(
-                    "[Execution]: Must be handled by sequential fallback: {:?}",
-                    e
-                )
+            Err(BlockExecutionError::FatalBlockExecutorError(PanicError::CodeInvariantError(err_msg))) => {
+                Err(VMStatus::Error { status_code: StatusCode::DELAYED_MATERIALIZATION_CODE_INVARIANT_ERROR, sub_status: None, message: Some(err_msg) })
```

Comment:
FatalBlockExecutionError happens only because of aggregators?

Reply:
aggregators or resource groups (i.e. delayed materialization)